carma-messenger-core/cpp_message/src/MobilityPath_Message.cpp

```diff
@@ -35,83 +36,26 @@ namespace cpp_message
   
         uint8_t buf[len];
         for(auto i=0;i<len;i++){
-            buf[i]=binary_array[i];
+            std::copy(binary_array.begin(),binary_array.end(),buf);
```

Comment:
This doesn't go in the for-loop it replaces the whole loop.

Reply:
Right, sorry.